README.md

```diff
@@ -27,153 +39,113 @@ two members. Thus we need to enable _sympathetic replication_, such that peers
+feeds when:
 
-_Diagram showing an example layout of group-related feeds. Note that the shards
-in your use-case will likely not be those shown, see how they are determined
-below._
+1. Creating a new group
+2. Adding a peer to a group
+3. Discovering group membership
 
-### 1. The additions feed
+### 3.1. The group/additions feed
 
 The purpose of this feed is to hold messages for coordination of joining groups.
+It represents the record of all peers the user had added to groups.
 
-- 1.1 Each peer running this spec MUST have an additions feed
-- 1.2 Each peer MUST deterministically place their additions feed as a subfeed,
-such that:
-  - Their additions feed is a direct subfeed of a shard feed
-  - The shard feed is a subfeed of the v1 subfeed, and derived from the string
-  `group/additions` according to the v1 tree structure specified in
+- 3.1.1 Each peer running this spec MUST have an additions feed
+- 3.1.2 Each peer MUST deterministically place their additions feed as a subfeed
+of a shard feed, such that:
+  - The shard feed is derived from the string "`group/additions`" according to
+  the v1 tree structure specified in
   [ssb-meta-feeds-spec].
   - The `metafeeds/add/derived` message announcing the additions feed
     - MUST have `feedpurpose` equal to the string `group/additions`,
     - MUST be of feed format `classic`
-    - MUST have `metadata` following the [ssb-meta-feeds-dm-spec] section 1
+    - MUST have `metadata` following the [ssb-meta-feeds-dm-spec] Section 1
     - MUST NOT be encrypted
-- 1.3 If a peer A wants to add another peer B in a group, then A MUST replicate
+- 3.1.3 If a peer A wants to add another peer B in a group, then A MUST replicate
 B's additions feed and B must replicate A's additions feed
-- 1.4 All messages published on the additions feed MUST be encrypted with [box2]
-encryption, also known as "envelope spec".
+- 3.1.4 All messages published on the additions feed MUST be "`group/add-member`"
+messages encrypted with [envelope-spec] encryption
   - See details below
-- 1.5 There MUST be at most one additions feed per metafeed tree
-  - This feed represents the record of all people joining each group and must
-  not be lost
+- 3.1.5 There MUST be at most one additions feed per metafeed tree
 
 #### `group/add-member` messages
 
 This is the only type of message currently expected in the additions feed.
-It's defined in the [private-group-spec] to look like this:
-
-```js
-{
-  type: 'group/add-member',
-  version: 'v2',
-  root: 'ssb:message/classic/THxjTGPuXvvxnbnAV7xVuVXdhDcmoNtDDN0j3UTxcd8=',
-  secret: '3YUat1ylIUVGaCjotAvof09DhyFxE8iGbF6QxLlCWWc=',
-  recps: [
-    'ssb:identity/group/vof09Dhy3YUat1ylIUVGaCjotAFxE8iGbF6QxLlCWWc=',    // group_id
-    'ssb:feed/bendybutt-v1/YXkE3TikkY4GFMX3lzXUllRkNTbj5E-604AkaO1xbz8=', // feed_id
-  ],
-  tangles: {
-    group: {
-      // ...
-    },
-    members: {
-      // ...
-    }
-  }
-}
```

Comment:
I would personally like this example message to stay. You can hide it in a details fold if you want.
I don't want us to correlate "dry and impenetrable" with "looks academic".

Note that your spec dropped the `version` field.

What this example captures is that it really bangs hold which sort of ssb-uri should b e used, and what's meant by "base64 encoded string". An alternative to this would be to store such examples as test-vectors and link to those from the text.

Reply:
summary:
- :fire: must add version
- <preference> would like example preserved in some way

Reply:
I don't understand why we *have* to add the version, I understand why we *should* add it but technically things are going to still work just fine if peers implement only what this spec is saying. Anyway, I'll just add it back, just because you're asking.

However, let's not block this PR on https://github.com/ssbc/private-group-spec/pull/23 because that one is not done yet, and we're trying to *remove* obstacles for milestone completion, not add obstacles.

Reply:
> What this example captures is that it really bangs hold which sort of ssb-uri should b e used, and what's meant by "base64 encoded string". An alternative to this would be to store such examples as test-vectors and link to those from the text.

I think we can just delegate this part of the spec to `private-group-spec`, so we don't need to "spec" it in two places.

Reply:
Well, whatever, I put version `v2` and `secret` instead of groupKey. Please work on https://github.com/ssbc/private-group-spec/pull/23 ASAP